composer.json

```diff
@@ -11,13 +11,19 @@
        "email": "jhedstrom@gmail.com"
      }
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/ademarco/DrupalDriver"
+    }
+  ],
```

Comment:
I've merged the changes to the DrupalDriver into both master and 1.0, so this change can be backed out.